src/components/question-editor/QuestionEditor.tsx

```diff
@@ -130,63 +156,108 @@ const QuestionEditor = () => {
     </Box>
   ))
 
+  const disabledButton =
+    type === 'openAnswer' ? text && openAnswer : text && answers[0]?.text
```

Comment:
better create an enum for types and also, you can use isOpenAnswer instead of type === 'openAnswer'

Reply:
Fixed